package.json

```diff
@@ -39,7 +39,8 @@
     "read": "1.0.7",
     "string-length": "1.0.1",
     "through2": "2.0.1",
-    "tmp": "0.0.28"
+    "tmp": "0.0.28",
+    "update-notifier": "^2.2.0"
```

Comment:
Just noticed this has the ^. Might want to freeze it to `"2.2.0"` in a different commit/PR.